src/react-distortable-imageoverlay-maplayer.js

```diff
@@ -1,7 +1,7 @@
 import { MapLayer, withLeaflet } from 'react-leaflet';
 import PropTypes from 'prop-types';
 
-import './lib/leaflet-distortableimage';
+import './node_modules/leaflet-distortable-image/dist/Leaflet.DistortableImage.js';
 import './lib/leaflet-path-transform';
 import L from 'leaflet';
 
```

Comment:
I can't get this compiling but I'm certain it's a problem with the way I've configured the project with `create-react-library`.  Thanks for the upstream changes.  I'll look at getting this in soon as I can.

Reply:
Awesome, this was just my guess... that's where npm would install the lib but I haven't tried running `npm install` to see if it actually shows up there. If you share your error maybe I can help debug!